project/dexter-monitor/public/ctrl/weeklyProjectCtrl.js

```diff
@@ -25,42 +25,41 @@
-        {field:'criFix',            width: 35,      headerCellTemplate:'<div style="text-align: center" uib-tooltip="[Fixed] Critical defects" tooltip-append-to-body="true">Cri<br>Fix</div>'},
-        {field:'criDis',            width: 35,      headerCellTemplate:'<div style="text-align: center" uib-tooltip="[Dismissed] Critical defects" tooltip-append-to-body="true">Cri<br>Dis</div>'},
-        {field:'majNew',            width: 35,      headerCellTemplate:'<div style="text-align: center" uib-tooltip="[New] Major defects" tooltip-append-to-body="true">Maj<br>New</div>'},
-        {field:'majFix',            width: 35,      headerCellTemplate:'<div style="text-align: center" uib-tooltip="[Fixed] Major defects" tooltip-append-to-body="true">Maj<br>Fix</div>'},
-        {field:'majDis',            width: 35,      headerCellTemplate:'<div style="text-align: center" uib-tooltip="[Dismissed] Major defects" tooltip-append-to-body="true">Maj<br>Dis</div>'},
-        {field:'minNew',            width: 35,      headerCellTemplate:'<div style="text-align: center" uib-tooltip="[New] Minor defects" tooltip-append-to-body="true">Min<br>New</div>'},
-        {field:'minFix',            width: 35,      headerCellTemplate:'<div style="text-align: center" uib-tooltip="[Fixed] Minor defects" tooltip-append-to-body="true">Min<br>Fix</div>'},
-        {field:'minDis',            width: 35,      headerCellTemplate:'<div style="text-align: center" uib-tooltip="[Dismissed] Minor defects" tooltip-append-to-body="true">Min<br>Dis</div>'},
-        {field:'crcNew',            width: 35,      headerCellTemplate:'<div style="text-align: center" uib-tooltip="[New] Coding rule checker defects" tooltip-append-to-body="true">Crc<br>New</div>'},
-        {field:'crcFix',            width: 35,      headerCellTemplate:'<div style="text-align: center" uib-tooltip="[Fixed] Coding rule checker defects" tooltip-append-to-body="true">Crc<br>Fix</div>'},
-        {field:'crcDis',            width: 35,      headerCellTemplate:'<div style="text-align: center" uib-tooltip="[Dismissed] Coding rule checker defects" tooltip-append-to-body="true">Crc<br>Dis</div>'},
-        {field:'etcNew',            width: 35,      headerCellTemplate:'<div style="text-align: center" uib-tooltip="[New] Etc defects" tooltip-append-to-body="true">Etc<br>New</div>'},
-        {field:'etcFix',            width: 35,      headerCellTemplate:'<div style="text-align: center" uib-tooltip="[Fixed] Etc defects" tooltip-append-to-body="true">Etc<br>Fix</div>'},
-        {field:'etcDis',            width: 35,      headerCellTemplate:'<div style="text-align: center" uib-tooltip="[Dismissed] Etc defects" tooltip-append-to-body="true">Etc<br>Dis</div>'}
+        {field:'year',              displayName:'Year',         width: '5%',    headerTooltip: 'Year'},
```

Comment:
As we discussed, what about setting horizaontal scroll bar for this page.
And give those columns minimum width, too.